### PR TITLE
feat: doors system

### PR DESCRIPTION
**Pull request by floydya**

### CODE REVIEW

**Comment by floydya** [comment]
Do not merge it please, I will test it properly and then will mark as "Ready for review".

**Comment by MnkyArts** [comment]
Thanks for integrating permissions! Looks good so far

File: src/main/client/webview/index.ts [hunk 1]
```diff
@@ -325,7 +325,11 @@ export function useWebview(path = 'http://assets/webview/index.html') {
     }
 
     function onSyncedMetaChange(object: alt.Object, key: string, newValue: any) {
-        if (native.isEntityAPed(object.scriptID) && object.scriptID === alt.Player.local.scriptID) {
+        if (
+            typeof object.scriptID === 'number' &&
```
Comment (floydya) [inline src/main/client/webview/index.ts:329]: Screenshot 2024-07-25 181123
Got this issue somehow, fixed with this line

File: src/plugins/dummy/server/index.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    if (!door) return;
+    doorController.toggleLockState(player, door.uid);
+});
+
+alt.on('playerConnect', async (player) => {
+    new alt.Vehicle('adder', spawnPosition.x, spawnPosition.y, spawnPosition.z, 0, 0, 0);
+    await alt.Utils.wait(1000);
+    player.model = 'mp_m_freemode_01';
+    player.spawn(spawnPosition);
+    Rebar.useKeybinder().updateKeybindForPlayer(player);
```
Comment (Stuyk) [inline src/plugins/dummy/server/index.ts:28]: extra file

**Review by Stuyk — CHANGES_REQUESTED** [review]

**Review by Stuyk — APPROVED** [review]

**Pull request merged**